File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.py
def solution(numbers):
    
    count = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
    
    for i in range(len(numbers)):
        count[numbers[i]] += 1
    
    result = 0
    for j in range(len(count)):
        if count[j] == 0:
            result += j
    
    return result

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

int solution(vector<int> numbers) {
    int answer = 0;
    vector<int> array;

    for(int i=0; i<numbers.size(); i++) {
        answer += numbers[i];
    }
    
    return 45-answer;
}